<?xml version="1.0"?>
<launch>
	<env name="ROSCONSOLE_CONFIG_FILE" value="$(find c2_hw)/config/c2_logger.config"/>
<rosparam ns="tool_wheel_pid">
l_pid: {p: 1.0, i: 0.2, d: 0.001,  v_max: 0.52, v_min: 0.1, i_err_max: 0.05 }
r_pid: {p: 0.8, i: 0.2, d: 0.0007, v_max: 0.52, v_min: 0.1, i_err_max: 0.05 }
plan:
  - {t: 10, l: -0.5, r: -0.5}
  - {t: 10, l:  0.5, r:  0.5}
</rosparam>
	<node name="tool_wheel_pid" pkg="c2_hw" type="tool_wheel_pid" output="screen">
		<remap from="/lwheel_pid" to="/c2/hardware/wheels/l_pid_args"/>
		<remap from="/rwheel_pid" to="/c2/hardware/wheels/r_pid_args"/>
		<remap from="/lwheel_vtarget" to="/c2/hardware/wheels/l_cmd_vel"/>
		<remap from="/rwheel_vtarget" to="/c2/hardware/wheels/r_cmd_vel"/>
		<remap from="/lwheel_state" to="/c2/hardware/wheels/l_vel"/>
		<remap from="/rwheel_state" to="/c2/hardware/wheels/r_vel"/>
	</node>
</launch>
